Menu/Menu.js

```diff
@@ -2,13 +2,32 @@
 const toggleMenu = () => {
   // Toggle the "menu--open" class on your menu refence. 
   menu.classList.toggle("menu--open")
+  
 }
 
 // Start Here: Create a reference to the ".menu" class
 const menu = document.querySelector(".menu");
+
 // create a reference to the ".menu-button" class
 const menuButton = document.querySelector(".menu-button");
 // Using your menuButton reference, add a click handler that calls toggleMenu
 menuButton.addEventListener("click", function(e) {
```

Comment:
There is an async case here between this anonymous function and toggleMenu(). Commenting out link 68 of index.css, fixes this issue.